### PR TITLE
Partner SKU Reviews Plugins

### PR DESCRIPTION
**Pull request by alistat**

### CODE REVIEW

File: src/plugins/partner_sku_reviews.coffee [hunk 1]
```diff
@@ -0,0 +1,1326 @@
...
+    <div class="sa-sales-wrap">
+      <img class="sa-sales-icon" alt="sales icon"/>
+      <span class="sa-sales-text">#{sales_text}</span>
+    </div>
+  """
+
+reviewsAggregation = (aggregation) ->
+  return '' unless aggregation && aggregation.length
+
+  aggregation_elements = aggregation.map ({label, score, style}) ->
```
Comment (kosdiamantis) [inline src/plugins/partner_sku_reviews.coffee]: [Minor] Let's add a comment for the supported styles, `sa-good`, `sa-so-so`, `sa-bad`

File: src/plugins/partner_sku_reviews.coffee [hunk 2]
```diff
@@ -0,0 +1,1326 @@
...
+  COLORS = ['white', 'grey', 'black', 'orange']
+  SIZES = ['small', 'medium']
+
+  settings = window.sa_plugins.settings
+  context = -> window.sa_plugins.partner_sku_reviews
+
+  asset_url = (source) -> "#{settings.url.base}/assets/#{source}"
+
+  constructor: ->
+    console.log(context())
```
Comment (kosdiamantis) [inline src/plugins/partner_sku_reviews.coffee]: I leave a comment, just to remember to remove this one. 😄

File: src/plugins/partner_sku_reviews.coffee (hunk 1 above)
Comment (alistat) [reply src/plugins/partner_sku_reviews.coffee]: Comments in general..

File: translations/alve.yml [hunk 3]
```diff
@@ -7,3 +7,21 @@ translations:
...
+    users_found_usefull: 'χρήστες τη βρήκαν χρήσιμη'
+    more: 'more'
+    at: 'at'
+    product_reviews: 'product reviews'
+    user_reviews: 'user reviews'
+    read_more: 'read more'
+    share_your_experience: 'Share your experience!'
+    write_review_for: 'Write a review for'
+    and_help: 'and help other users!'
+    review_this: 'Review this product'
```
Comment (kosdiamantis) [inline translations/alve.yml]: add a newline

File: src/plugins_manager.coffee [hunk 4]
```diff
@@ -13,27 +13,31 @@ define [
...
     ###
     notify: (action, data) ->
       @_fetchEnabledPlugins().then =>
-        plugin = PluginsSettings.triggers[action]
-        return unless plugin
+        plugins = PluginsSettings.triggers[action]
+        return unless plugins
 
-        return if @_pluginIsLoaded(plugin) || !@_pluginIsEnabled(plugin)
+        plugins = [plugins] unless Object.prototype.toString.call(plugins) == '[object Array]' # single plugin specified as string
```
Comment (kosdiamantis) [inline src/plugins_manager.coffee]: [Minor] I suggest to add a comment with a link for the polyfill: [mozilla/Array/isArray#Polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#Polyfill)

**Pull request merged**